docs/source/cli/sawtooth.rst

```diff
@@ -0,0 +1,282 @@
+..
+   Copyright 2017 Intel Corporation
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+.. _sawtooth-cli-reference-label:
+
+************
+Sawtooth CLI
+************
+
+sawtooth
+========
+
+.. literalinclude:: output/sawtooth_usage.out
+   :language: console
+   :linenos:
+
+sawtooth admin
+==============
+
+.. literalinclude:: output/sawtooth_admin_usage.out
+   :language: console
+   :linenos:
+
+sawtooth admin genesis
+======================
+
+Overview
+--------
+
+The genesis CLI tool produces a file for use during initialization of a
+validator.  A network requires an initial block (known as the genesis block)
+whose signature will determine the block chain id.  This initial block is
+produced from a list of batches, which will be applied at genesis time.  The
+input to the command is a set of zero or more files containing serialized
+``BatchList`` protobuf messages. The output is a file containing a serialized
+``GenesisData`` protobuf message.  This file, when placed at
+``<sawtooth_data>/genesis.batch``, will trigger the genesis process.
+
+The location ``sawtooth_data`` depends on whether or not the environment
+variable ``SAWTOOTH_HOME`` is set.  If it is, then ``sawtooth_data`` is located
+at ``<SAWTOOTH_HOME>/data``.  If it is not, then ``sawtooth_data`` is located
+at ``/var/lib/sawtooth``.
+
+Usage
+-----
+
+.. literalinclude:: output/sawtooth_admin_genesis_usage.out
+   :language: console
+   :linenos:
+
+Arguments
```

Comment:
Should "Arguments" and "Output" be omitted since they are part of the automation?